sw/device/sca/aes_serial.c

```diff
@@ -564,14 +584,75 @@ static void aes_serial_fvsr_key_batch_encrypt(const uint8_t *data,
   }
   sca_set_trigger_low();
 
-  // Only send the first word to increase capture rate
-  aes_send_ciphertext(true);
+  // Acknowledge command
+  simple_serial_send_status(0);
+
+  aes_send_ciphertext(false);
```

Comment:
This change seems unrelated to the this commit/PR. Is it intentional? If yes, please factor it out into a separate commit and document it. Because the corresponding change also needs to be done in ot-sca.

Reply:
This is now factored into a separate commit.